Module3_2/Module3_2/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace Module3_2
+{
+    class Program
+    {
+        static void Main()
+        {
+            int naturalNumberFromUser = GetNumber();
+            int amountOfNaturalEvenNumbers = GetAmountOfNaturalEvenNumbers();
```

Comment:
you dont need 2 input values, according to task, you need to get 1 (let it be X) input natural number and show in the output first X even numbers from infinity row of natural numbers